### PR TITLE
Codesplittingreactselect

### PR DESCRIPTION
**Pull request by sneridagh**

### CODE REVIEW

File: cypress/integration/blocks-image.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ if (Cypress.env('API') !== 'guillotina') {
       cy.waitForResourceToLoad('@breadcrumbs');
       cy.waitForResourceToLoad('@actions');
       cy.waitForResourceToLoad('@types');
-      cy.waitForResourceToLoad('?fullobjects');
+      cy.waitForResourceToLoad('my-page?fullobjects');
```
Comment (tisto) [inline cypress/integration/blocks-image.js:13]: @sneridagh doesn't that wait for "/my-page/my-page?fullobjects"? I mean we are on "mypage/edit", right?

File: cypress/support/commands.js [hunk 2]
```diff
@@ -11,12 +11,14 @@ Cypress.Commands.add('autologin', () => {
     password = 'secret';
   }
 
-  cy.request({
-    method: 'POST',
-    url: `${api_url}/@login`,
-    headers: { Accept: 'application/json' },
-    body: { login: user, password: password },
-  }).then(response => cy.setCookie('auth_token', response.body.token));
+  return cy
```
Comment (tisto) [inline cypress/support/commands.js:14]: @sneridagh just out of curiosity. Could you elaborate why returning cy.request here makes a difference?

Comment (sneridagh) [reply cypress/support/commands.js:14]: It seems that Cypress expects you to return (the promise) so he can know when it’s resolved better.

File: cypress/integration/blocks-image.js (hunk 1 above)
Comment (sneridagh) [reply cypress/integration/blocks-image.js:13]: I left it like this to be more explicit to know what are we expecting but later on, I realised the command is using an ‘includes’ so it does not make much of a difference.

**Pull request merged**